### PR TITLE
Fixed security group exporter command for Kilo neutron cli

### PR DESCRIPTION
**Pull request by ajvb**

### CODE REVIEW

File: openstack/migrate-helpers/osmigrator.rb [hunk 1]
```diff
@@ -108,15 +108,15 @@ def secgroup_export
         importer_script.puts("# Start of #{secgroup_name} Rules")
 
         # Add the command to create the security group.
-        sec_description = group_details[0]["Value"]
+        sec_description = group_details["description"]
```
Comment (computermode) [inline openstack/migrate-helpers/osmigrator.rb:111]: It's cleaner now :D

**Comment by jfrost** [comment]
+1

**Comment by computermode** [comment]
+1, can't wait to use it

File: openstack/migrate-helpers/osmigrator.rb [hunk 2]
```diff
@@ -108,15 +108,15 @@ def secgroup_export
...
         if sec_description.nil?
           importer_script.puts("neutron security-group-create #{secgroup_name}")
         else
           importer_script.puts("neutron security-group-create #{secgroup_name} --description \"#{sec_description}\"")
         end
 
         # Parse out the actual rules. It requires a little bit of hackery because its not actually json.
-        rules = JSON.parse('[' + group_details[3]["Value"].gsub(/}\n/, "},") + ']')
+        rules = JSON.parse('[' + group_details["security_group_rules"].gsub(/}\n/, "},") + ']')
```
Comment (mcasper) [inline openstack/migrate-helpers/osmigrator.rb:119]: Heh, whatever format they return here is absolutely hilarious. I tried to come up with a better way to do this and came to the same exact solution that you did.

Oh openstack CLIs...

**Comment by mcasper** [comment]
+1

File: openstack/migrate-helpers/osmigrator.rb (hunk 2 above)
Comment (ajvb) [inline openstack/migrate-helpers/osmigrator.rb:119]: Word, and then trying to use the HTTP API is straight torture.

**Pull request merged**